spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -605,9 +598,18 @@ def test_np_threshold(kernel):
         assert "{'float_kind': <built-in method format of str object" in content
 
 
+TURTLE_ACTIVE = False
+try:
+    import turtle
+    turtle.Screen()
+    turtle.bye()
+    TURTLE_ACTIVE = True
+except:
+    pass
```

Comment:
Please move this block to be where the `TKINTER_INSTALLED` one was present. We prefer to declare constants on top and not in the middle of tests.

Reply:
I put it there, because if running interactively, it pops up a window to test for this, and I thought it made more sense to do this immediately before the test it relates to. But I'm happy to move it if you prefer.

Reply:
Ok, I see. Then please add a comment on top of this block explaining that.

Reply:
Done!